### PR TITLE
Update run.sh to match documentation

### PR DESCRIPTION
**Pull request by iandroogmans**
The mqtt_port env variable doesnt have a default value, while documentation states it's not mandatory.

P.S. thanks for the efforts on this project!

### CODE REVIEW

File: run.sh
Comment (thomasddn) [inline run.sh]: Please start comments with a capital

File: run.sh
Comment (thomasddn) [inline run.sh]: Please start comments with a space and a capital, gateway in lowercase

**Comment by thomasddn** [comment]
Good catch and thanks for fixing it straight away! I've added two small remarks, could you fix those? I like a unified coding style, but you could not know since there is no contribution info. ;)

I'm waiting on some other issue before I will merge and release this one. I hope to receive news in the coming week.

**Comment by iandroogmans** [comment]
Gotcha, should be correct now.

**Review by thomasddn — APPROVED** [review]

**Pull request merged**
